### ISSUE TITLE
Translation / Localization

### ISSUE THREAD

**Issue by tschach** [listsfordesign/Lists#37]
Hi there,

thanks for the effort on this project! Any plans to have these lists translated to other languages (e.g. headlines, jobs) or localized to other countries (e.g. colleges, dates) where appropriate? Would be glad to help - in my case for a German version.

Best
Tobias

**Comment by AxelTheGerman** [comment]
localization would be great. I'd recommend having the locale as part of the file name or in the folder structure so it can be read programmatically, e.g. `addressgermany.json` should be `address_de.json` or `de/address.json` or `address/de.json` (leaning towards the first option. but depending on the number of lists and languages, a folder structure might be good - or can be by logical categories)

The naming scheme should be simple but for for both, languages and locations. For example the `airports_us.json` (US being the 2 letter country code) `quotes_de.json` (DE being the locale). Also should the default one be no code (being English) or `_en` to be explicit?

Edit: a more programmer oriented approach is this project - https://github.com/stympy/faker/tree/master/lib/locales

**Comment by tschach** [comment]
Just saw that there are localized addresses and names already - not sure why I did not see this at first... would be interesting to hear from @jperriere and others whether translations (and more localizations) would fit the basic idea/concept or rather be an overkill.

**Comment by vltrajvlien** [comment]
Hi,

Thanks both of you for your interest in contributing to the Lists. I'm on vacation with limited access to internet, sorry I didn't get back to you sooner.

The vision I have for this project is that we cover the most use cases possible. So the most content we have, the better. As you saw I already started to make some lists country-specific : [address](https://github.com/tschach/Lists/blob/master/Lists/addressgermany.json) and [names](https://github.com/tschach/Lists/blob/master/Lists/namesgermany.json).

I think the smarter way to go about it is to make it country-specific and not language-specific. For example, Portugal and Brasil both speak portuguese but don't necesarily have the same address format. For the sake of simplicity I'd rather have all the lists in one folder. So we could experiment  with a naming convention like this : `address_de.json`.

tl;dr : Thanks for your help, go 👍 
Julien
